cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -129,7 +129,8 @@ std::vector<std::unique_ptr<::ray::TaskArg>> TransformArgs(
     } else {
       RAY_CHECK(arg.id);
       ray_arg = absl::make_unique<ray::TaskArgByReference>(ObjectID::FromBinary(*arg.id),
-                                                           ray::rpc::Address{});
+                                                           ray::rpc::Address{},
+                                                           /*call_site=*/"");
```

Comment:
Since we probably want to add actor repr/name as well as call site, I wonder if we need to pass the struct instead?  Like `struct refMetadata`? so that we can easily add new metadata fields.

Reply:
Let's do this in a follow-up PR once we actually have other metadata.